packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -1,10 +1,21 @@
 import Web3 from "web3";
-const { isAddress } = Web3.utils;
+const { isAddress, toChecksumAddress, toBN } = Web3.utils;
 
 import assert from "assert";
 
 import type { RateModel } from "@uma/financial-templates-lib";
 
+// Check each token rate model contains the expected data.
+const expectedRateModelKeys = ["UBar", "R0", "R1", "R2"];
+
+// const supported L2 Chain IDS:
+const supportedChainIds = [
+  10, // optimism mainnet
+  69, // optimism testnet
+  42161, // arbitrum mainnet
+  421611, // arbitrum testnet
+];
```

Comment:
Why hardcode this? Should this list be an optional config param instead?

Reply:
in the future, yes. In the future, the bot can figure out which chain IDs are whitelisted on the contract and then run there. Right now, though, to move as fast as possible, we don't do any auto-chainId detection and force the user to provide